src/helpers/analytics.js

```diff
@@ -2,6 +2,12 @@ import { HiiveAnalytics, HiiveEvent } from "@newfold-labs/js-utility-ui-analytic
 import { HIIVE_ANALYTICS_CATEGORY } from "../constants";
 
 export const trackHiiveEvent = (action, data) => {
+	// Check if the label key for the event is present and not empty
+	const labelKey = data.label_key;
+	if (labelKey && !data[labelKey]) {
+		return;
```

Comment:
@mr-vara Wouldn't this obscure the issue from developers? Shouldn't we at least have a console.error() or something?

Reply:
@wpscholar I'll add console.error with a proper pattern and required data.